### PR TITLE
Add libcdms for UV-CDAT

### PR DESCRIPTION
**Pull request by dnadeau4**

### CODE REVIEW

**Comment by dnadeau4** [comment]
@ocefpaf When you have time...  :smile:

File: recipes/libcdms/build.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    --with-pnglib=/usr/X11/lib \
+    --with-grib2lib=${PREFIX}/lib \
+    --with-jasperlib=${PREFIX}/lib \
+    --with-grib2inc=${PREFIX}/include \
+    --enable-grib2 \
+    --prefix=${PREFIX}
+make 
+make libinstall
+make bininstall
+# no make check or make test
```
Comment (ocefpaf) [inline recipes/libcdms/build.sh]: :+1:

File: recipes/libcdms/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  sha256: 5456bfa2ac3b5ff7d0d05fce7ac2da3d2cdf08017a4f4019c99ada91f7e48649
+
+build:
+  number: 0
+  skip: True  # [win] 
+
+requirements:
+  build:
+    - hdf5
+    - libnetcdf
```
Comment (ocefpaf) [inline recipes/libcdms/meta.yaml]: Please you pin the dependencies according to https://github.com/conda-forge/staged-recipes/wiki/Pinned-dependencies

Comment (dnadeau4) [inline recipes/libcdms/meta.yaml]: :+1:

**Pull request merged**